src/ontology/mondo-edit.obo

```diff
@@ -258177,6 +258191,7 @@ id: MONDO:0013154
 name: muscular dystrophy-dystroglycanopathy (congenital with brain and eye anomalies), type A2
 def: "An autosomal recessive muscular dystrophy caused by mutations in the POMT2 gene. It is associated with characteristic brain and eye malformations and profound mental retardation." [NCIT:C126742]
 synonym: "MDDGA2" EXACT ABBREVIATION [MONDO:Lexical, OMIM:613150]
+synonym: "muscle-eye-brain-POMT2 related" EXACT CLINGEN_PREFERRED []
```

Comment:
could you please add a dbxref here:
https://clinicalgenome.org/affiliation/50061/